templates/umami/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+services:
+  umami:
+    image: ghcr.io/umami-software/umami:postgresql-latest
```

Comment:
Can we use defined version? this in order to prevent breaks templates in the future

Reply:
Good catch, fixing